docs/static/releasenotes.asciidoc

```diff
@@ -30,6 +31,78 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-8-1]]
+=== Logstash 7.8.1 Release Notes
+
+==== Performance improvements and notable issues fixed
+
+===== Fixed performance regression during pipeline compilation
+
+This regression was introduced in 7.7.0 and described in the release notes of 7.8.0 ***TODOLINK[HERE]***.
+Introducing the logging of the plugin id caused the id to explicity end up in the generated code during compilation, causing an explosion of generated classes and zero reuse of the cache.
+By changing the assignment of plugin.id to load the value dynamically instead, this has now been fixed. Find more details in https://github.com/elastic/logstash/pull/12038[#12038]
+
+===== Multiple fixes to pipeline shutdown and reloading
+
+7.8.1 brings many fixes that improve the stability of the pipeline lifecycle. These include avoiding overlapping state resolution operations when monitoring was enabled (https://github.com/elastic/logstash/pull/12034[#12034]), ensuring that the persistent queue lock is correctly released if a pipeline fails to start https://github.com/elastic/logstash/pull/12023[#12023], and avoid a known JRuby issue by not spawning a Ruby thread from Java code https://github.com/elastic/logstash/pull/11900[#11900].
+
+===== Changes in the dependencies of Beats/TCP/HTTP Input Plugins
+
+To avoid breaking changes and retain control on the set of ciphers exposed in these plugins the `tcnative` + `boringssl` dependencies have been dropped in favor of using JVM supplied ciphers instead.
+Overall you should see no impact but this may result in fewer ciphers being available if the JCE unlimited strength jurisdiction policy is not installed.
+NOTE: this policy is installed by default on versions of the JDK from u161 onwards
+You can find more information in the release notes of the respective plugins.
+
+==== Plugins
+
+*Xml Filter - 4.1.1*
+
+* Fix: exceptions thrown while handling events no longer crash the pipeline https://github.com/logstash-plugins/logstash-filter-xml/pull/73[#73]
+
+*Beats Input - 6.0.11*
+
+* Updated jackson databind and Netty dependencies. Additionally, this release removes the dependency on `tcnative` +
+      `boringssl`, using JVM supplied ciphers instead. This may result in fewer ciphers being available if the JCE
```

Comment:
The `+` sign in `tcnative`+`boringssl` is being treated as an asciidoc continuation character in the bulletetd plugin sections. It looks like `tcnative`-`boringssl` is a popular alternative. (Unless the`-` literally means `minus` in those cases.

Other possible alternatives are `tcnative+boringssl` or `tcnative` `+` `boringssl`. This affects beats, http, and tcp input sections. If you don't like any of those, we can work together to come up with more.

Reply:
"tcpnative and boringssl" ?